### PR TITLE
Sentry frontend - setup

### PR DESCRIPTION
**Pull request by jankrzempek**
https://trello.com/c/6HcaCcjp/1786-add-sentry-monitoring-for-front-end-errors

Mechanism for storing error from Frontend. Please visit https://docs.google.com/document/d/1UVJbkcClyip9WlVy-92LAbNdSIXC0k6y3uZ_xlRTBuA/edit#heading=h.bhaof14chkdu if you would like to log in.

To pass an error to Sentry, use the LogController. It wraps up the Sentry code and provides an interface for error handling.

### CODE REVIEW

File: app/javascript/packs/react-app.js [hunk 1]
```diff
@@ -1,10 +1,24 @@
 import { createRoot } from "react-dom/client";
 import React from "react";
+import * as Sentry from "@sentry/react";
 
 import { App } from "../react/App.tsx";
 
 import "../react/assets/styles/typography";
 
+Sentry.init({
+    environment: process.env.NODE_ENV === "production" ? "Producation" : "Development",
```
Comment (smialko) [inline app/javascript/packs/react-app.js:10]: `"Producation"` ?

**Review by smialko — CHANGES_REQUESTED** [review]
One typo but probably important :D

File: app/javascript/packs/react-app.js (hunk 1 above)
Comment (jankrzempek) [reply app/javascript/packs/react-app.js]: Right, thanks!

**Review by smialko — APPROVED** [review]
code looking good but at the same time when I'm reviewing it i feel like doggo. So yeah, approved. 
Screenshot 2024-05-29 at 12 40 56

**Review by evulins — APPROVED** [review]
🥇

**Pull request merged**